gitlab/manifest.json

```diff
@@ -16,8 +17,8 @@
     "source control",
     "issue tracking"
   ],
-  "doc_link": "https://docs.datadoghq.com/integrations/gitlab/",
+  "metric_prefix": "gitlab.", // is this valid see metadata file?
```

Comment:
is this comment resolved?

Reply:
No, otherwise it wouldn't be there. The idea is to answer the question or give some insight that could help.

Reply:
Keeping the prefix to `gitlab.` we don't have gitlab dashboards